apps/web/src/pages/Communities/IndexCommunityPage/components/CommunityTable/CommunityTable.tsx

```diff
@@ -101,10 +100,7 @@ export const CommunityTable = ({
     .filter(notEmpty);
 
   const { roleAssignments } = useAuthorization();
-  const canCreateMembers = checkRole(
-    [ROLE_NAME.PlatformAdmin],
-    roleAssignments,
-  );
+  const canCreateMembers = hasRole([ROLE_NAME.PlatformAdmin], roleAssignments);
```

Comment:
Another instance where we could use a permission constant

Reply:
Yeah, added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12007/commits/8e0598ea3565b544d8f1f0e50221c082e7bc2707

Reply:
Permission name I added does not match variable name, instead I matched it to what it controls 

![image](https://github.com/user-attachments/assets/3de1b661-a360-4de6-a7a6-4617da56bee1)